### DISCUSSION TITLE
Add ODBC adapter

### DISCUSSION THREAD

**Discussion by rupurt** [tconbeer/harlequin#370] [Ideas (Feature Requests)]
Howdy,

Love the project! Are there any plans to add an ODBC adapter? e.g. with https://github.com/mkleehammer/pyodbc. That would give Harlequin the ability to connect to nearly any database.

Cheers

**Comment by tconbeer** [comment]
Yep I'm in favor. Going to make this an issue

**Comment by tconbeer** [comment]
Released yesterday! https://github.com/tconbeer/harlequin-odbc

**Reply by rupurt** [reply]
Amazing! Thank you.

Quick attempt I'm running into numpy errors on Linux but will debug tonight

**Reply by rupurt** [reply]
@tconbeer made some progress 😄. Currently running into an error when trying to create the config file https://github.com/tconbeer/harlequin-odbc/issues/4
